diagnosis/getkuberneteslogs.sh

```diff
@@ -134,15 +269,48 @@ else
     || { echo "The identity file $IDENTITYFILE is not a RSA Private Key file."; echo "A RSA private key file starts with '-----BEGIN [RSA|OPENSSH] PRIVATE KEY-----''"; exit 1; }
 fi
 
+if [ "$UPLOAD_STORAGE_ACCOUNT"=="Y" ]; then
```

Comment:
Not sure this message is necessary. Script does not comment about the other parameters unless there is an issue